sentry/src/main/java/io/sentry/Span.java

```diff
@@ -52,6 +56,9 @@ public SentryTraceHeader toSentryTrace() {
   @Override
   public void finish() {
     this.timestamp = DateUtils.getCurrentDateTime();
+    if (this.throwable != null) {
+      hub.setSpanContext(this.throwable, this);
+    }
```

Comment:
just a `nit`, we don't need to use `this.` if there's no class/parameters shadowing